moped-database/docker-compose.yml

```diff
@@ -3,7 +3,7 @@
 version: '3.7'
 services:
     hasura:
-        image: hasura/graphql-engine:v2.34.0
+        image: hasura/graphql-engine:v2.35.1
```

Comment:
this will not actually bump the Hasura engine version in production. that is controlled by ECS config :)